src/foam/lib/csv/CSVEscapeStringParser.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.lib.csv;
+
+import foam.lib.parse.*;
+
+
+public class CSVEscapeStringParser implements Parser {
+  public final static char ESCAPE = '"';
+
+  public CSVEscapeStringParser() {
+  }
+
+  public PStream parse(PStream ps, ParserContext x) {
+    if ( ps == null ) return null;
+    char head = ps.head();
+    if ( head != ESCAPE ) return null;
+
+    int delimCount = 0;
+    StringBuilder sb = builders.get();
+    ps = ps.tail();
+
+    while ( ps.valid() ) {
+      head = ps.head();
+      if ( head == ESCAPE ) {
+        if ( delimCount == 1) {
+          sb.append(ESCAPE);
```

Comment:
Do you really want to append the ESCAPE character to the string buffer?

It looks like a string `"foo"` will parse to `"foo"` and `foo` will parse to `foo`

But I think we'd rather have `foo` and `"foo"` both parse to `foo`

Reply:
"foo" will parse to foo
the first " will not be appended, same as the last one